nautobot/ipam/tests/test_views.py

```diff
@@ -303,16 +303,14 @@ def test_prefix_ipaddresses_table_list_includes_child_ips(self):
             status=ip_status,
             namespace=self.namespace,
         )
-        url = reverse("ipam:prefix_ipaddresses", args=(instance.pk,))
+        url = reverse("ipam:prefix_ip_addresses", args=(instance.pk,))
```

Comment:
Consider changing the `url_name` on the DistinctViewTab to avoid this - though more correct and consistent, it's technically a breaking change.